Validation/RecoMuon/python/customiseForNewMuonValidation.py

```diff
@@ -0,0 +1,109 @@
+import FWCore.ParameterSet.Config as cms
+
+def enableNewMuonVal(process):
```

Comment:
if the idea is to run either the "old" or "new", you would make things much cleaner if the plugin names did not get prepended with "new". Lots of the code duplication would go away that way